rpcserver.go

```diff
@@ -3080,6 +3090,7 @@ func createRPCOpenChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 		LocalChanReserveSat:   int64(dbChannel.LocalChanCfg.ChanReserve),
 		RemoteChanReserveSat:  int64(dbChannel.RemoteChanCfg.ChanReserve),
 		StaticRemoteKey:       dbChannel.ChanType.IsTweakless(),
```

Comment:
Is `StaticRemoteKey` also true for anchors?

Reply:
yes

Reply:
Isn't that confusing? We could also define the bool as indicating that particular format, not a format feature.

Reply:
True, it could be either or. Not sure how much it matters, as we are deprecating it.

Reply:
Indeed, it doesn't matter much. My preference would be to consider the boolean a channel type and not a channel feature, because it is simpler to explain. `ListChannels` only communicates channel types.

Reply:
Went with your suggestion :)